src/os/os.c

```diff
@@ -624,6 +622,18 @@ long long os_get_time_monotonic(void)
     time = system_get_rtc_time();
     time *= clockmul;
     time >>= 2;
+#elif defined(CONFIG_IDF_TARGET)
```

Comment:
Possibly not, I remember seeing some inconsistencies in the usage of `clock_gettime` and that is why this was added some time back (originally wrote this code in Mar/Apr 2020). I've done some additional testing with this block removed and it does appear to work but needs a minor fix to prevent overflow in the time calculation just a few lines after this block (break up the `1000000000LL` similar to the `__WIN32__` block). I'll clean this up before this PR is ready for merge.